app/src/app-logic.js

```diff
@@ -1,16 +1,22 @@
-import React, { useCallback, useMemo, useState } from 'react'
-import { useAragonApi, useConnectedAccount } from '@aragon/api-react'
+import React, { useCallback } from 'react'
+import {
+  AragonApi,
+  useApi,
+  useAppState,
+  useCurrentApp,
+} from '@aragon/api-react'
```

Comment:
There's a few more exports (that I just realized we forgot to document) that simplify how we use `@aragon/api-react` here :).